test/unit/three_scale/analytics/user_tracking_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'segment/analytics'
```

Comment:
Why do we need this require?

Reply:
Since here (https://github.com/3scale/porta/pull/903/files#diff-757942a1b9ceda38bc0b76acea39c446R175) we expected that `Segment::Analytics` never receives the action method, we need to require it, otherwise it fails since it doesn't have it loaded yet.

Reply:
It works for me 🤔 
![image](https://user-images.githubusercontent.com/11318903/59765527-92fc3100-929e-11e9-97da-abe4d6c7b29a.png)

Reply:
yeah.. worked for me.. I don't know why it failed the first time haha.. thanks @Martouta